ui/src/components/chat/sidebar.rs

```diff
@@ -314,6 +343,11 @@ pub fn Sidebar(cx: Scope<Props>) -> Element {
                     let other_participants =  state.read().remove_self(&participants);
                     let user: state::Identity = other_participants.first().cloned().unwrap_or_default();
                     let platform = user.platform().into();
+                    let is_group_conv = participants.len() > 2;
+                    let is_admin = match chat.creator.clone() {
```

Comment:
I have a pet peeve about `clone()`. I prefer to avoid it if possible, though using it here is probably not an issue at all. Still, you could use `chat.creator.as_ref().map(|x| x == &state.read().did_key()).unwrap_or_default()`